libs/galley-types/src/Galley/Types.hs

```diff
@@ -867,6 +879,7 @@ instance FromJSON Member where
         Member <$> o .:  "id"
                <*> o .:? "service"
                <*> o .:? "otr_muted"        .!= False
+               <*> o .:? "otr_muted_status"
```

Comment:
[unrelated to this PR, but perhaps easy to fix]

what's the difference between `MemberUpdate` and `MemberUpdateData`?  why do we need `ToJSON` instances for these?  can you explain this in haddocks near the code?

Reply:
`MemberUpdateData` is used for events (sent over the websocket, etc.) while `MemberUpdate` is what is expected by `galley` on the endpoint.

That being said, thanks for the question since it brought up an incomplete implementation on my end!